packages/core/src/components/step-by-step/step-by-step.css

```diff
@@ -0,0 +1,96 @@
+.hds-step-by-step__container {
+  position: relative;
+}
+
+/**
+ * Correct the font size and margin on `h1` elements within `section` and
+ * `article` contexts in Chrome, Firefox, and Safari.
+ */
+
+.hds-step-by-step__container h1 {
+  font-size: 2em;
+  margin: 0.67em 0;
+}
+
+.hds-step-by-step__steps-container {
+  --title-font-size: var(--fontsize-heading-m);
+  --title-line-height: var(--lineheight-xl);
+
+  counter-reset: item;
+  margin: 0;
+  padding: 0;
+}
+
+ol.hds-step-by-step__steps-container {
+  --circle-size: 48px;
+  --step-left-padding: 96px;
+}
+
+.hds-step-by-step__step-item:before,
+.hds-step-by-step__step-item:after {
+  box-sizing: border-box;
+  content: '';
+  display: block;
+  left: calc(var(--circle-size) / 2 + var(--step-left-padding) / 2);
+  position: absolute;
+  top: calc(var(--spacing-l) + (var(--title-font-size) * var(--title-line-height) / 2) - (var(--circle-size) / 2));
+}
+
+.hds-step-by-step__step-item:after {
+  background-color: #fff;
+  border: 3px solid #000;
+  border-radius: 50%;
+  height: var(--circle-size);
+  transform: translateX(-100%);
+  width: var(--circle-size);
+  z-index: 2;
+}
+
+ol.hds-step-by-step__steps-container .hds-step-by-step__step-item:after {
+  --line-height: 1.3;
+
+  align-items: center;
+  content: counter(item);
+  counter-increment: item;
+  display: flex;
+  font-size: 1.25rem;
+  font-weight: 500;
+  justify-content: center;
+  line-height: var(--line-height);
+}
+
+.hds-step-by-step__step-item {
+  background-color: var(--color-coat-of-arms-light);
+  list-style: none;
+  margin: var(--spacing-l) 0;
+  padding: var(--spacing-l) var(--spacing-l) var(--spacing-l) var(--step-left-padding);
+  position: relative;
+}
+
+ol.hds-step-by-step__steps-container .hds-step-by-step__step-item {
+  padding: var(--spacing-l) var(--spacing-l) var(--spacing-l) var(--step-left-padding);
+}
+
+ul.hds-step-by-step__steps-container {
+  --circle-size: 18px;
+  --step-left-padding: var(--spacing-5-xl);
+}
+
+.hds-step-by-step__step-item__title {
```

Comment:
Not valid BEM, would suggest `.hds-step-by-step__step-item-title` instead 🙂

Reply:
Good catch! [7350611](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1004/commits/7350611781dd18270282d2e885a378f7eba18c87)